pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -26,6 +26,24 @@ self: super: {
   # Doesn't compile with lua 5.2.
   hslua = super.hslua.override { lua = pkgs.lua5_1; };
 
+  # Please also remove optparse-applicative special case from
+  # cabal2nix/hackage2nix.hs when removing the following.
+  elm-make = super.elm-make.override {
+    optparse-applicative = optparse-applicative_0_10_0;
+  };
+  elm-package = super.elm-package.override {
+    optparse-applicative = optparse-applicative_0_10_0;
+  };
+  # elm-compiler jail-break can be removed after next elm-compiler
+  # release: bumped language-ecmascript's limit in git already.
+  elm-compiler = doJailbreak super.elm-compiler;
+
+  safecopy = dontCheck super.safecopy;
```

Comment:
It would be great to inform the package author about the test suite failures at http://hydra.cryp.to/build/357301/nixlog/1/raw and to add a comment here that refers to the ticket, if the guy happens to use Github.